cypress/e2e/movieListPage.cy.ts

```diff
@@ -0,0 +1,101 @@
+describe("MovieListPage", () => {
+  // beforeEach(() => {
+  //   cy.visit("http://localhost:3000/");
+  // });
+
```

Comment:
The same for almout all testcases - need to check that exact movie exist on the page - that is the final resut. Not the tiles length